troubleshooting.md

```diff
@@ -429,3 +428,17 @@ You'll need to either:
 or as root if your user has not enough privileges.
 
 * sudo loginctl enable-linger $UID
+
+### 18) `podman run` fails with "bpf create: permission denied error" 
```

Comment:
There is an extra space at the end of this line that is causing the tests to fail. Remove the space and repush and we should be able to merge.

Reply:
nice catch @rhatdan